packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -36,6 +36,9 @@
 	// as align-items: stretch is inherited by default.Additionally due to older browser's flex height
 	// interpretation, any percentage value is likely going to cause issues, such as metaboxes overlapping.
 	// See also https://www.w3.org/TR/CSS22/visudet.html#the-height-property.
+	&.has-surround {
```

Comment:
I think this should move the `EditorCanvas` component within the "editor" package and should be based on whether there's a "goBack" setting or something, that way we unify between post and site editors.